Formula/openjdk.rb

```diff
@@ -97,29 +124,28 @@ def install
         --with-x=#{HOMEBREW_PREFIX}
         --with-cups=#{HOMEBREW_PREFIX}
         --with-fontconfig=#{HOMEBREW_PREFIX}
+        --with-freetype=system
```

Comment:
This seems to be bundled on macOS too. Does using the system version not work?

Reply:
`openjdk` only allows bundled copy on macOS https://github.com/openjdk/jdk19u/blob/jdk-19+36/make/autoconf/lib-freetype.m4#L128-L132
```m4
  if test "x$FREETYPE_TO_USE" = "xsystem" && \
     (test "x$OPENJDK_TARGET_OS" = "xwindows" || \
     test "x$OPENJDK_TARGET_OS" = "xmacosx"); then
    AC_MSG_ERROR([Only bundled freetype can be specified on Mac and Windows])
  fi
```